### PR TITLE
feat: add `operations` test runner

### PR DESCRIPTION
**Pull request by phklive**
**Motivation**

Added the complete`operations` test runner

**Description**

Added the `operations` test runner that enables us to test the functionality of our core internal functions as `process_execution_payload` and others.

### CODE REVIEW

File: native/libp2p_nif/main.h [hunk 1]
```diff
@@ -0,0 +1,117 @@
+/* Code generated by cmd/cgo; DO NOT EDIT. */
+
+/* package command-line-arguments */
+
+
```
Comment (MegaRedHand) [inline native/libp2p_nif/main.h:5]: Please remove this file.

File: test/spec/runners/operations.ex [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+  end
+
+  @doc """
+  Each handler has associated types, hence we resolve each type that has to be used from each handler
+  """
+  def resolve_type_from_handler(handler) do
+    case Map.get(@type_map, handler) do
+      nil -> raise "Unknown case #{handler}"
+      type -> Module.concat(SszTypes, type)
+    end
```
Comment (MegaRedHand) [inline test/spec/runners/operations.ex:209]: [`Map.fetch!`](https://hexdocs.pm/elixir/1.12.3/Map.html#fetch!/2) can also be used for this

**Review by MegaRedHand — APPROVED** [review]

File: native/libp2p_nif/main.h (hunk 1 above)
Comment (phklive) [reply native/libp2p_nif/main.h]: Removed

File: test/spec/runners/operations.ex (hunk 2 above)
Comment (phklive) [reply test/spec/runners/operations.ex]: Would it be better using Map.fetch! ?

Comment (MegaRedHand) [reply test/spec/runners/operations.ex]: I think it'd be worse, because the message is less clear (i.e. "key ... not found in: %{...}").

**Pull request merged**